src/collar/oc_exceptions.lsl

```diff
@@ -181,7 +183,7 @@ ExMenu(key kID, string sWho, integer iAuth) {
         iExSettings = g_iOwnerDefault;
     else if (sWho == "trusted" || ~llListFindList(g_lSecOwners, [sWho]))
         iExSettings = g_iTrustedDefault;
-    if (~iInd = llListFindList(g_lSettings, [sWho])) // replace deefault with custom
```

Comment:
Were these assignments to iInd and iNames being relied on anywhere?

Reply:
no they were not